### ISSUE TITLE
Missing centralized_workflow.png on /book/en/v2/Distributed-Git-Distributed-Workflows page

### ISSUE THREAD

**Issue by axmbo** [git/git-scm.com#540]
There's a missing image on /book/en/v2/Distributed-Git-Distributed-Workflows page. It's alt text is <figcaption>Centralized workflow.</figcaption>.

**Comment by pwolanin** [comment]
Just noticed the same - not a good bug when it's among the 1st pages a new reader sees

**Comment by hatzopoulos** [comment]
It would seem the image missing from [the page](http://git-scm.com/book/en/v2/Distributed-Git-Distributed-Workflows) has been renamed to [centralized.png](https://git-scm.com/book/en/v2/book/05-distributed-git/images/centralized.png).

![Centralized Workflow](https://git-scm.com/book/en/v2/book/05-distributed-git/images/centralized.png)

*verified with page 152 of the [pdf version](https://progit2.s3.amazonaws.com/en/2015-09-27-d6ba8/progit-en.844.pdf)*

**Comment by YueLinHo** [comment]
I think it's fixed by PR #652 and can be closed.

**Comment by pedrorijo91** [comment]
duplicated from #661 

already solved @peff
